brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/spec/RelationshipModifier.java

```diff
@@ -37,7 +41,10 @@ public void apply(EntitySpec<?> entitySpec, NodeTemplate nodeTemplate, Topology
             RelationshipTemplate relationshipTemplate = findRelationshipRequirement(nodeTemplate, requirementId);
             if (relationshipTemplate != null && relationshipTemplate.getType().equals("brooklyn.relationships.Configure")) {
 
-                Map<String, Object> relationProperties = getTemplatePropertyObjects(relationshipTemplate);
```

Comment:
Unfortunately, there are subtle differences that make that unworkable.  `relationshipTemplate` is passed in here instead of the `nodeTemplate`, while the `PaaSNodeTemplate` and `Map<String, PaaSNodeTemplate>` are derived from the `nodeTemplate`.